workstation/w_darwin.go

```diff
@@ -8,40 +8,26 @@ import (
 	"strings"
 
 	log "github.com/Sirupsen/logrus"
-	"github.com/xshellinc/tools/constants"
 	"github.com/xshellinc/tools/dialogs"
 	"github.com/xshellinc/tools/lib/help"
 	"github.com/xshellinc/tools/lib/sudo"
 )
 
-// Darwing specific workstation type, contains unMount and diskUtil strings
-// that are used to perform operations on mounted disks
-type darwin struct {
-	*workstation
-	*unix
-	unMount  string
-	diskUtil string
-}
+const diskUtil = "diskutil"
 
-// Initializes darwing workstation with unix and darwin specific program names
+// Initializes default workstation
 func newWorkstation() WorkStation {
 	m := new(MountInfo)
 	var ms []*MountInfo
-	w := &workstation{runtime.GOOS, true, m, ms}
-	ux := &unix{constants.UnixDD, constants.MountDir, constants.Umount, constants.Eject}
-
-	return &darwin{
-		workstation: w,
-		unix:        ux,
-		unMount:     constants.DarwinUmount,
-		diskUtil:    constants.DarwinDiskutil}
+
+	return &workstation{runtime.GOOS, true, m, ms}
 }
 
 const diskSelectionTries = 3
 const writeAttempts = 3
 
-// Notifies user to chose a mount, after that it tries to write the data with `diskSelectionTries` number of retries
-func (d *darwin) WriteToDisk(img string) (job *help.BackgroundJob, err error) {
+// Notifies user to choose a mount, after that it tries to write the data with `diskSelectionTries` number of retries
+func (d *workstation) WriteToDisk(img string) (job *help.BackgroundJob, err error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 21 of function (*workstation).WriteToDisk() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->